charms/worker/k8s/requirements.txt

```diff
@@ -17,3 +17,5 @@ websocket-client==1.8.0
 poetry-core==1.9.1
 lightkube==0.16.0
 httpx==0.27.2
+loadbalancer_interface == 1.2.0
+cryptography==44.0.0
```

Comment:
OH dude, i have bad news.  You can't use this library b/c that library requires our charm not be able to support multiple series.  

`cryptography` compiles C code specific for the ubuntu base's python.  Building on focal, yields code that doesn't run on jammy or noble.  The same is true if you build on jammy -- it doesn't run on focal/noble. 

my best advice.  subprocess out to `openssl` on the device